.travis.yml

```diff
@@ -1,10 +1,10 @@
 sudo: required
+services:
+  - xvfb
 language: java
 
 jdk:
-  - oraclejdk8
-  - oraclejdk9
-  - openjdk7
+  - oraclejdk11
   - openjdk11
```

Comment:
add openjdk8 as well please

Reply:
The reason I removed these is because older versions fail with: `Expected feature release number in range of 9 to 17, but got: 8`